packages/wix-ui-core/package.json

```
@@ -138,7 +138,7 @@
     "react-testing-library": "^6.0.3",
     "slash": "^2.0.0",
     "ts-jest": "^22.0.3",
-    "typescript": "~3.0.3",
+    "typescript": "^3.7.4",
```

I'd suggest to keep the tilde
```suggestion
    "typescript": "~3.7.4",
```